tools/headers.ts

```diff
@@ -1,6 +1,208 @@
 import { writeFile, stat } from "node:fs/promises";
 import { z } from "zod";
 
+/** @link https://chatgpt.com/c/6795dae3-8a10-800e-96af-fd0d01579f39 */
+const responseOnlyHeaders = {
+  "accept-patch": {
+    proof:
+      "Defined in RFC 5789, Section 3.1. Used by the server to indicate supported PATCH media types.",
+    reason:
+      "Servers use this header in responses to inform clients about supported PATCH formats.",
+  },
+  "accept-post": {
+    proof: "Part of the WebDAV specification (RFC 7240, Section 4).",
+    reason:
+      "Sent in responses to indicate the types of POST requests that a server supports.",
+  },
+  "accept-ranges": {
+    proof: "Defined in RFC 7233, Section 2.3.",
+    reason:
+      "The server uses this to tell the client it supports partial requests (e.g., byte ranges).",
+  },
+  age: {
+    proof: "Defined in RFC 7234, Section 5.1.",
+    reason:
+      "Indicates the age of a cached response, a concept relevant only to responses.",
+  },
+  allow: {
+    proof: "Defined in RFC 7231, Section 7.4.1.",
+    reason: "Used in responses to indicate supported methods for the resource.",
+  },
+  "alt-svc": {
+    proof: "Defined in RFC 7838, Section 3.",
+    reason: "The server advertises alternative services in responses.",
+  },
+  "cache-status": {
+    proof: "Defined in RFC 9211.",
+    reason: "Indicates the status of caching for the response.",
+  },
+  "cdn-cache-control": {
+    proof:
+      "An extension header often used by CDNs to communicate caching strategies; see examples in CDN-specific docs (e.g., Akamai or Cloudflare).",
+    reason: "Relevant to response caching.",
+  },
+  "cdn-loop": {
+    proof: "Defined in RFC 8586, Section 2.",
+    reason:
+      "Used to detect infinite loops within CDNs; appears only in responses.",
+  },
+  "content-base": {
+    proof:
+      "Mentioned in early HTTP/1.1 specs (e.g., RFC 2068, Section 14.11). Deprecated, but applies only to server responses.",
+    reason: "Specifies the base URL for relative URLs in a document.",
+  },
+  "content-disposition": {
+    proof: "Defined in RFC 6266, Section 4.",
+    reason:
+      "Specifies how content is displayed or downloaded, relevant only in responses.",
+  },
+  "content-encoding": {
+    proof: "Defined in RFC 7231, Section 3.1.2.2.",
+    reason: "Indicates compression or encoding applied to the response body.",
+  },
```

Comment:
not sure about these, I saw them in file uploads
https://github.com/mscdex/busboy/blob/6b3dcf69d38c1a8d53a0b3e4c88ba296f6c91525/test/test-types-multipart.js#L850

Reply:
```suggestion
```